# Messages
bool cwdirection = True
float32 radius = 1.0
float32 lin_vel = 1.0
